ClosedXML/Excel/Protection/IXLWorkbookProtection.cs

```diff
@@ -1,6 +1,13 @@
-﻿namespace ClosedXML.Excel
+﻿// Keep this file CodeMaid organised and cleaned
+using System;
+using static ClosedXML.Excel.XLProtectionAlgorithm;
+
+namespace ClosedXML.Excel
 {
     public interface IXLWorkbookProtection : IXLElementProtection<XLWorkbookProtectionElements>
     {
+        IXLWorkbookProtection Protect(Algorithm algorithm, XLWorkbookProtectionElements allowedElements);
+
+        IXLWorkbookProtection Protect(String password, Algorithm algorithm = DefaultProtectionAlgorithm, XLWorkbookProtectionElements allowedElements = XLWorkbookProtectionElements.Windows);
```

Comment:
I was thinking that instead of having multiple overloads with different numbers of parameters, we should probably move toward having a single methdod with options object (e.g. `Protect(XLProtectOptions options)`). This will make adding additional parameters easier, as well as this would better support backward compatibility between different versions of the library. As a drawback, the default values will be a bit more obscure. 
What do you think?

Reply:
Ah, I see now that there still would be multiple overloads of `Protect` method because of the inherited interfaces. Nevermind then.